ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -867,21 +868,38 @@ module.exports = function CesiumMap(
           options.isSelected ? -1 : 0
         )
       } else if (geometry.constructor === Cesium.Label) {
-        const geometryPosition = geometry.position
-        // finds an existing label that has the same position; returns undefined if none
-        const labelWithSamePosition = _.find(
-          mapModel.get('labels'),
-          label =>
-            label.position.x === geometryPosition.x &&
-            label.position.y === geometryPosition.y
-        )
-        // if there is an existing label with the same position, then this one
-        // should not be displayed
-        if (
-          labelWithSamePosition !== undefined &&
-          geometry.id !== labelWithSamePosition.id
-        ) {
-          geometry.show = false
+        geometry.isSelected = options.isSelected
```

Comment:
Don't see this referenced anywhere else in the code. Does it need to be assigned?

Reply:
It's referenced in line 949 of this file when determining which label to show after clustering is turned off.